src/main/java/oripa/domain/paint/byvalue/ByValueContext.java

```diff
@@ -4,16 +4,16 @@
  * Copyright (C) 2005-2009 Jun Mitani         http://mitani.cs.tsukuba.ac.jp/
 
     This program is free software: you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
```

Comment:
The "public" in comments should not be removed.
I know putting a license comment in every code is an old way and annoying, though.

Reply:
Sorry, my oversight. I'll fix this in a moment.